---
string active_map
map_manager/Map[] maps
bool success
string message